setup.cfg

```diff
@@ -81,6 +81,7 @@ format = pylint
 ignore =
     E731  # do not assign a lambda expression, use a def
     W504  # line break occurred after a binary operator
+    # TODO: delete these next 3 because we no longer use black
     E203  # whitespace before ':'. Opposite convention enforced by black
     E231  # missing whitespace after ',', ';', or ':'; for black
     W503  # line break before binary operator, need for black
```

Comment:
```suggestion
    W503  # line break before binary operator
    W504  # line break after binary operator
    # TODO: delete these next 2 because we no longer use black
    E203  # whitespace before ':'. Opposite convention enforced by black
    E231  # missing whitespace after ',', ';', or ':'; for black
```